application.rb

```diff
@@ -0,0 +1,184 @@
+require_relative 'services/login_service.rb'
+require_relative 'services/sign_up_service.rb'
+require_relative 'services/task_manager.rb'
+
+class Application
+  @@TaskManager = nil 
+
+  def self.boot
+    @@TaskManager = TaskManager.new
+    start_event_loop 
+  end
+
+  def self.assign_task(user)
+    puts 'enter assignee user id'
+    assignee_user_id = gets.chomp
+    puts 'Enter Description about task'
+    description = gets.chomp
+    puts 'Enter Due_date Format (YYYY/MM/DD)'
+    due_date = gets.chomp
+    puts 'Enter priority eg: 1, 2, 3'
+    priority = gets.chomp
+    @@TaskManager.add_task(
+      assignee_user_id: assignee_user_id,
+      description: description,
+      due_date: due_date,
+      priority: priority, 
+      user: user
+    )
+  end
+
+  def self.delete_task
+    puts "Enter Task id you want to delete"
+    delete_id = gets.chomp.to_i
+    @@TaskManager.delete_task(delete_id)
```

Comment:
`task_manager.delete(task_id)`
1. fix TaskManager as suggested by Neeraj.
2. `delete` method should not contain `task` as task manager should delete tasks only, not users :)
3. let this be `task_id` rather than `delete_id`. `delete_id` is inaccurate variable naming because it doesn't explain what we're deleting.